test/test_cli.py

```diff
@@ -13,7 +13,36 @@
 WHOAMI_RESPONSE = {
     "account": "myaccount"
 }
+VAR_HELP = '''Usage:
+     conjur [global options] variable <subcommand> [options] <VARIABLE_ID> <VALUE>
```

Comment:
I would refrain from writing out the whole usage here. Maybe
```
'''Usage:
    conjur
```
is enough because if we change the UX of the help page, we also will have to change the tests which is not sustainable.

You can then use `assertRegex` or `assertIn` instead of `assertEquals`